### ISSUE TITLE
Cannot build switch example for Android because "switch is a Java keyword"

### ISSUE THREAD

**Issue by t-arn** [beeware/toga#1162]
**Describe the bug**
I wanted to build the switch example for Android and get a build error.

**To Reproduce**
1. cd examples\switch
2. briefcase create android
3. briefcase build android

This produces following error:
```
> Task :app:processDebugResources FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Package 'org.beeware.switch' from AndroidManifest.xml is not a valid Java package name as 'switch' is a Java keyword.
```

**Expected behavior**
The example should build without errors.

**Environment:**
 - Operating System: Windows
 - Python version: 3.8
 - Software versions:
   - Briefcase: 0.3.3
   - Toga: 0.3.0.dev25

**Comment by freakboy3742** [comment]
Thanks for the report. Ironically, this was reported against briefcase as beeware/briefcase#460 - but the logical follow up of fixing the project name here was never done.

**Comment by t-arn** [comment]
The issue has been solved with PR #1168
